File: Python/boj/Silver/7795.py
import sys

if __name__ == "__main__":
    try:
        sys.stdin = open("input.txt", "r")
    except FileNotFoundError:
        pass

    T = int(input())
    case = list(sys.stdin.read().split("\n"))
    result = []

    for i in range(T):
        count = 0
        A = list(map(int, case[i * 3 + 1].split(" ")))
        B = sorted(list(map(int, case[i * 3 + 2].split(" "))), reverse=True)

        for a in A:
            for b in B:
                if a > b:
                    count += len(B) - B.index(b)
                    break

        result.append(count)

    print("\n".join(map(str, result)))

File: JavaScript/boj/Silver/7795.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
const [T, ...input] = fs.readFileSync(filePath).toString().trim().split('\n');

const result = [];

const getCount = (A, B) => {
  let count = 0;

  A.forEach((num) => {
    for (let i = 0; i < B.length; i++) {
      if (num > B[i]) {
        count += B.length - i;
        break;
      }
    }
  });

  return count;
};

for (let i = 0; i < T; i++) {
  const A = input[i * 3 + 1].split(' ').map(Number);
  const B = input[i * 3 + 2]
    .split(' ')
    .map(Number)
    .sort((a, b) => b - a);

  result.push(getCount(A, B));
}

console.log(result.join('\n'));
